drill-cicd-gradle-plugin/src/main/kotlin/com/epam/drill/integration/gradle/Extensions.kt

```diff
@@ -93,17 +87,36 @@ open class BaselineExtension(
     var targetRef: String? = null,
 )
 
-abstract class AgentExtension(
+open class AgentExtension(
+    var enabled: Boolean? = null,
+
     var version: String? = null,
     var downloadUrl: String? = null,
     var zipPath: String? = null,
 
     var logLevel: String? = null,
     var logFile: String? = null,
 
-    var additionalParams: Map<String, String>? = null
+    var additionalParams: Map<String, String> = mutableMapOf()
```

Comment:
Some additionalParams are of type `boolean` (e.g. `withJsCoverage: true`)
I know we're not doing JS agent right now, but can we still allow for boolean params too?

Reply:
It is still possible to pass a boolean parameter, you just need to take it in quotes:
```
additionalParams = [devToolsProxyAddress: "http://localhost:9090", withJsCoverage: "true"]
```
In any case, we will have to serialize the values ​​into a string.